common/dependabot-common.gemspec

```diff
@@ -35,8 +35,6 @@ Gem::Specification.new do |spec|
   spec.add_dependency "parser", ">= 2.5", "< 4.0"
   spec.add_dependency "toml-rb", ">= 1.1.2", "< 3.0"
 
-  spec.add_development_dependency "debase", "0.2.3"
-  spec.add_development_dependency "debase-ruby_core_source", "0.10.16"
```

Comment:
This dependency was added in https://github.com/dependabot/dependabot-core/pull/5086/

Had to remove it because `debase` doesn't currently support Ruby 3 or 3.1: https://github.com/ruby-debug/ruby-debug-ide/issues/223#issuecomment-1009609192

Ruby 3 ships with a new `debug` gem? don't have many details on it... 

Not sure if we should also remove `ruby-debug-ide` or at least bump the version pin? https://github.com/dependabot/dependabot-core/pull/5086/files#diff-e843e646a7311fb218cf05845818b31d84bc04c0742f0567395c0c289cacf72cR46

cc @pavera as he may have more context since he wrote the PR originally adding this.

Reply:
> Ruby 3 ships with a new debug gem? I don't have many details on it...

It's in the [ruby org](https://github.com/ruby/debug), and we use it in the Bundler ecosystem ([Bundler 1](https://github.com/dependabot/dependabot-core/blob/main/bundler/helpers/v1/Gemfile#L7) and [Bundler 2](https://github.com/dependabot/dependabot-core/blob/main/bundler/helpers/v2/Gemfile#L7)). I think it makes sense to keep it (I see it's declared on the next line down from this diff).

Reply:
I added this to add support for debugging rspec tests in vscode.  We saw issues where later versions of debase had issues, I believe with missing pre-compiled binary blobs, hence the version pin.  It's easy enough to move forward and see if things behave so I'd say bump it forward and I'll look at any resulting errors.

Reply:
Thanks guys, I have a few followup questions:
1. Does this `debug` gem have to be pinned now that it's part of the stdlib?
2. Should I remove the `ruby-debug-ide` gem altogether? It's unclear to me from the gem readme if it's used by the new stdlib `debug` gem or if it's only used for `debase`... and if the latter, then should be removed altogether since `debase` was removed.

I'll reach out on slack/zoom for clarification... as I had some broader surrounding context questions due to my ruby ignorance.

Reply:
My recollection is that `ruby-debug-ide` brought in `debase` as a dependency.  `ruby-debug-ide` is the "magic" that allows vscode debugging.  Since debase is not yet Ruby 3.1, and ruby-debug-ide hasn't had any updates lately, I think we'll have to remove all of these and figure out the new debugger.

Reply:
Deleted `ruby-debug-ide` per ☝️.

Will sync with Landon shortly to see if we can drop the pin for the `debug` gem now that it's in the 3.1 `stdlib`.